style_homepage.css

```diff
@@ -0,0 +1,774 @@
+{
+    background-color: black;
+    width: 100%;
+    padding-left: 20%;
+    padding-bottom: 5%;
+}
+
+.work_01_second_div img  /*Properties of second div of sub-container with projects section*/
+{
+    max-width: 60%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.flex_container_work_02 /*Sub-container 02 with Projects*/
+{
+    display: flex;
+    width: 100%;
+    height: 70%;
+    padding-top: 2%;
+    padding-bottom: 3%;
+    font-size: 1.5rem;
+    color: #fffdf0;
+    background-color: black;
+}
+
+.flex_container_work_03 /*Sub-container 03 with Projects*/
+{
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    background-color:black;
+    text-align: center;
+}
+
+.flex_container_work_03_first_div /*First div of sub-container 03 with Projects*/
+{
+    width: 100%;
+    text-align: center;
+}
+
+.flex_container_work_03_first_div img /*Properties of first div of sub-container 03 with projects section*/
+{
+    max-width: 90%;
+}
+
+.flex-container_footer /*Properties of footer section*/
+{
+    text-align: center;
+    background-color: #db9cc0;
+    width: 100%;
+    height: 70%;
+    padding-top: 3%;
+    
+}
+p{
+    text-align: center;
+    font-size: 2.5rem;
+    color: #fffdf0;
+}
+
+.flex-container_footer_second_div
+{
+    text-align: center;
+    font-size: 1.5rem;
+    color: #fffdf0;
+    padding-bottom: 5%;
+    font-weight: lighter;
+}
+
+/*Media Query for Mobile - Global styling*/
+
+@media (max-width: 600px)
+{
+    .flex_container_me
+    {
+        display: flex;
+        height: 150px;
+        flex-direction: column;
+    }
+    .flex_container_work
+    {
+    display:flex;
+    width:60%;
+    flex-direction: column;
+    border: 1px black solid;
+    }
+.flex_container_work_01, .flex_container_work_02
+    {
+    display: flex;
+    height:50%;
+    flex-direction: column;
+    }
+}
+
+/*Contact page styling*/
+
+.container /*Contact form main container*/
+{
+    padding-top: 5%;
+    width: 80%;
+    padding: 10%;
+    background-color:#db9cc0;
+}
+
+.contact-box
+{
+    background:#fffdf0;
+    width: 100%;
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.contact-left{
+    flex-basis: 70%;
+    padding: 7% 0;
+}
+
+.input-row{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
+
+}
+.input-row .input-group{
+    flex-basis: 45%;
+}
+input{
+    width: 100%;
+    border: none;
+    border-bottom: 1px solid #ccc;
+    outline: none;
+    padding: 5%;
+}
+textarea{
+    width: 100%;
+    border: 1px solid #ccc;
+    outline: none;
+    padding: 10px;
+    box-sizing: border-box;
+}
+
+label{
+
+    margin-bottom: 6px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1rem;
+    display: block;
+    color: #af7c99;
+}
+
+button{
+background:#db9cc0;
+width: 8rem;
+border: none;
+outline: none;
+color: #fff;
+height: 3rem;
+border-radius: 30px;
+margin-top: 20px;
+box-shadow: 0px 5px 15px 0px rba(28,0,181,0.3);

```

Comment:
Where did this code come from? It is not taking effect in the website because `rga` is not a valid CSS function.

But the point is that this should be explained or a source cited.